dicom-archive/updateMRI_Upload.pl

```diff
@@ -235,6 +237,9 @@ =head1 AUTHORS
 );
 $db->connect();
 
+my $xxxxx = &NeuroDB::MRI::scan_type_text_to_id('scout', $db);
+my $yyyyy = &NeuroDB::MRI::scan_type_id_to_text(52, $db);
```

Comment:
not sure I understand what those two lines are for. Maybe you added them for testing MRI.pm and forgot to remove them? If so, I guess the use NeuroDB::MRI is not necessary as well in line 103.

Reply:
Arghh!!! The perils of issuing a PR at 10PM...I'll remove this